apisix/discovery/nacos/init.lua

```diff
@@ -208,7 +207,7 @@ local function de_duplication(services, namespace_id, group_name, service_name,
 end
 
 
-local function iter_and_add_service(services, values)
+local function iter_and_add_service(services, values,nacos)
```

Comment:
```suggestion
local function iter_and_add_service(services, values, nacos)
```
there are still problems like this in your PR, please fix all of them. You can use your IDE's formatting tool to fix such errors (VSCode has a tool already)

Reply:
OK I'will try